src/scout_apm/api/__init__.py

```diff
@@ -44,3 +44,50 @@ def tag(self, key, value):
         if self.span is not None:
             self.span.tag(key, value)
 
+
+class Transaction(ContextDecorator):
+    def __init__(self, name, tags={}):
+        self.name = name
+        self.tags = tags
+
+    def start(kind, operation, tags={}):
```

Comment:
Is this a `@classemthod`? Should be `start(cls, kind, operation, tags={})`?

Reply:
Shrug, I'll add it, but it's complexity that we aren't using.